lessons/Algebra2/exploring-logarithmic-relationships/langs/en-us/index.adoc

```diff
@@ -0,0 +1,163 @@
+
+=== Launch
+
+@center{@image{images/wealth-v-life-linear.png}}
+
+There definitely appears to be a relationship here, but it's not linear. Using `lr-plot` only shows a moderate correlation, because the straight line doesn't really fit the curve. The shape of this starts out rising fast on the left, before tapering off as `pc-gdp` increases to the right. What functions behave like this?
+
+=== Investigate
+
+The shape of this relationship is _logarithmic_, meaning it can be modeled using a logarithmic function:
+
+@center{   @math{f(x) = b \times log(x) + c}    }
+
+We need to find @math{b} and @math{c}, such that the model fits the data as closely as possible.
+
+Pyret has a few functions that may come in handy here:
+
+- @show{(contract "num-log" '("Number") "Number" "Consumes a number, and produces its natural log")}
+- @show{(contract "num-log-base" '("Number" "Number") "Number" "Consumes a number and a base, and produces the log of that number with the base")}
+- @show{(contract "num-expt" '("Number" "Number") "Number" "Consumes a number and a power, and produces that number raised to the power")}
+
+[.lesson-instruction]
+Complete @printable-exercise{model-wealth-v-lifespan.adoc}
+
+=== Synthesize
+
+- What was the best @math{R^2} value you found?
+- What points did you use?
+- What, if any, other strategies did you use to come up with your model?
+
+== Using Log Transformations @duration{25 minutes}
+
+=== Overview
+
+Students learn to transform the data into a linear shape, building a new column by applying a function to each row. This new data can be fit with a linear model. By applying the _inverse_ of this transformation to their linear model, they retrieve the quadratic model.
+
+=== Launch
+
+Deriving a logarithmic model from only two points is problematic. How do we know that those two points are representative of the trend? If we picked a wealthy country that is at war, we'd find an artificially low life expectancy. A model based on that point probably not fit the rest of the points very well!
+
+This is the same problem we ran into with linear models in our state demographics data, where a model built from any two points would only be as good as those points were representative.
+
+Trial-and-error only gets us so far, and it's not clear that we would ever stumble upon the optimal model. We need something like Pyret's `lr-plot` function, which uses computational methods to find the best possible model. Unfortunately, `lr-plot` only finds linear models, so we'd need to find a transformation that will make this data appear linear in order to use it.
+
+Imagine that the scatter plot is printed on a sheet of rubber, and can be stretched or squished in any way we want. Data Scientists often use *transformations* to stretch their data into shapes that are easier to use, and then reverse the transformation when they are done.
+
+[.lesson-instruction]
+- Complete questions 1-6 in @printable-exercise{transforming-wealth.adoc}
```

Comment:
Maybe add a discussion question: How was entering 'albania["pc-gdp"] / 2' similar to entering transform-pc-gdp(albania)? How was it different? 

Would also add (indented + italicized) answers to both of these questions.